packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -0,0 +1,248 @@
+package io.flutter.plugins.imagepicker;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.media.MediaScannerConnection;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.support.v4.app.ActivityCompat;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+public class ImagePickerDelegate
```

Comment:
Can you please add some do comments to the public members of this class to make the lives of future developers working on this code easier?

Reply:
What are the most painful parts that would need documenting? I find it to be pretty _self-documenting_ (heh), but that might be because I've refactored and stared at this code a lot. :)

Reply:
How about at least one overall doc comment for the entire class?